packages/react-instantsearch-hooks/src/__tests__/useSortBy.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { createInstantSearchTestWrapper } from '../../../../test/utils';
+import { useSortBy } from '../useSortBy';
+
+const getSortByTestItems = () => [
+  { label: 'Featured', value: 'indexName' },
+  { label: 'Price (asc)', value: 'indexName_price_asc' },
+  { label: 'Price (desc)', value: 'indexName_price_desc' },
+];
```

Comment:
Why do you need to return the items from a function?

Reply:
Ah right I also wondered that, forgot to post the comment in the end.

Reply:
I (wrongly 😅 ) assumed the assertion would not check for deep equality, and I wanted to prevent sharing the same reference in the test. I checked jest and it indeed does deep comparison, I'll update the test accordingly.